website/index.html

```diff
@@ -4,118 +4,130 @@
+            <input class="submit" type="submit" value="Search">
+        </form>
+        <div class="downloads">
+            <img class="downloads_content" src="img/app-store-badge.svg" alt="#">
+            <img class="downloads_content" src="img/google-play-badge.svg" alt="#">
+        </div>
+    </section>
+   
+    <!--       Offer      -->
+    <section class="offer_section">
+        <h2 class="offer_title">What do we offer</h2>
+        <div class="line"></div>
+        <div class="offer_container">
+           <div class="offer_item">
+                <svg class="offer_item_support"><use xlink:href="#Support"></use></svg>
+                <p class="offer_item_text">Support 24/7</p>
             </div>
-            <div class="item">
-                <div class="item_wrapper">
-                    <img class="item_icon" src="img/Real guest.png">
-                </div>
-                <div class="item_title">Real guest reviews</div> 
+            <div class="offer_item">
+                <svg class="offer_item_communicate"><use xlink:href="#Communicate"></use></svg>
+                <p class="offer_item_text">Communicate directly</p>  
+            </div>               
+            <div class="offer_item">
+                <svg class="offer_item_book"><use xlink:href="#Book"></use></svg>
+                <p class="offer_item_text">Book online</p>  
+            </div>    
+            <div class="offer_item">
+                <svg class="offer_item_real_guest"><use xlink:href="#Real"></use></svg>
+                <p class="offer_item_text">Real guest reviews</p>
             </div>
+            
         </div>
-    </div>
+    </section>
         <!--       Footer      -->
-        
-        <div class="footer">
-            <ul class="footer_column">
-                <li class="footer_title">About</li>
-                <li class="footer_text">How Triphouse works</li>
-                <li class="footer_text">Careers</li>
-                <li class="footer_text">Privacy</li>
-                <li class="footer_text">Terms</li>
-            </ul>
-            <ul class="footer_column">
-                <li class="footer_title">Property types</li>
-                <li class="footer_text">Guest houses</li>
-                <li class="footer_text">Hotels</li>
-                <li class="footer_text">Apartments</li>
-                <li class="footer_text">Villas</li>
-                <li class="footer_text">Holiday homes</li>
-                <li class="footer_text">Hostels</li>    
-            </ul>
-            <ul class="footer_column">
-                <li class="footer_title">Support</li>
-                <li class="footer_text">Contact Customer Service</li>
-                <li class="footer_text">FAQ</li>
-            </ul>
-            <div class="copyright_text">&#169; 2020 Triphouse, Inc. All rights reserved</div>
+    <footer class="footer">
```

Comment:
pls remove and don't use this class. you can address to footer directly by `footer {}`